src/crimson/os/futurized_store.cc

```diff
@@ -10,20 +10,20 @@
 
 namespace crimson::os {
 
-std::unique_ptr<FuturizedStore>
+seastar::future<std::unique_ptr<FuturizedStore>>
```

Comment:
Why do we need to futurize the method? In this patch can't see any place taking advantage of that. Is a follow-up coming in?

Reply:
Yes, a follow-up is coming, I'm working on a segment manager for ZNS devices. We want to be able to auto-detect whether a device is a ZNS device in make_seastore() in seastore.cc, which requires us to futurize it, and FuturizedStore::create() uses make_seastore() so we have to futurize it too, and it's a quick fix. I'll edit the PR description to reflect this.